frontend/src/lib/k8s/api/v2/hooks.ts

```diff
@@ -55,6 +55,14 @@ export interface QueryResponse<DataType, ErrorType> {
 export interface QueryListResponse<DataType, ItemType, ErrorType>
   extends QueryResponse<DataType, ErrorType> {
   items: Array<ItemType> | null;
+  /**
+   * Results from individual clusters. Keyed by cluster name.
+   */
+  clusterResults?: Record<string, QueryListResponse<DataType, ItemType, ErrorType>>;
```

Comment:
is clusterResults ever used? I'd think that items list with all the responses is enough? 

this is a bit hard to review without seeing other changes

Reply:
> this is a bit hard to review without seeing other changes to the ListViews of resources

You can see clusterErrors used here in this PR from the other day:
https://github.com/headlamp-k8s/headlamp/pull/2436